Libplanet/Net/Swarm.cs

```diff
@@ -2150,12 +2150,24 @@ private async Task ProcessFillTxs(CancellationToken cancellationToken)
                         $"Some tasks faulted during {nameof(GetTxsAsync)}().");
                 }
 
-                foreach (var task in tasks.Where(task => !task.IsFaulted))
+                foreach (Task<List<Transaction<T>>> task in tasks.Where(task => !task.IsFaulted))
                 {
+                    // `task.Result` is okay because we've already waited.
                     txs.AddRange(task.Result);
                 }
 
-                BlockChain.StageTransactions(txs.ToImmutableHashSet());
+                foreach (Transaction<T> tx in txs)
+                {
+                    try
+                    {
+                        BlockChain.StageTransaction(tx);
+                    }
+                    catch (InvalidTxException ite)
+                    {
+                        _logger.Error(ite, "{tx} will not be staged since it is invalid.", tx);
```

Comment:
As `Transaction<T>` does not override `ToString()` method, it would not have much information.  Printing its ID would be rather better.

```suggestion
                        _logger.Error(ite, "{TxId} will not be staged since it is invalid.", tx.Id);
```